src/Interface.lua

```diff
@@ -160,7 +160,7 @@ function Cool.UI.Update(setKey)
         set.onCooldown = false
         label:SetText("")
         texture:SetColor(1, 1, 1, 1)
-        Cool.UI.PlaySound(Cool.preferences.sets[setKey].sounds.onReady)
+        if Cool.preferences.audioNotifications then Cool.UI.PlaySound(Cool.preferences.sets[setKey].sounds.onReady) end
```

Comment:
What do you think about moving the condition to check for `audioNotifications` moving to inside `Cool.UI.Playsound`?

e.g.
```
function Cool.UI.PlaySound(sound)
    if sound.enabled and Cool.preferences.audioNotifications then
        PlaySound(SOUNDS[sound.sound])
    end
end
```